app/assets/stylesheets/refreshments/_invoice.scss

```diff
@@ -54,6 +52,13 @@ $copy-icon-color: #fff;
     position: relative;
   }
 
+  &__price {
+    &--loading {
+      color: transparent;
+      text-shadow: 0 0 5px rgba(0,0,0,0.5);
```

Comment:
Expected single space after "," in a single-line function   function-comma-space-after<br>Unexpected leading zero                                     number-leading-zero